<?xml version="1.0"?>

<launch>
  <arg name="frame"/>
  <arg name="worldFrame" default="world"/>

  <node name="controller" pkg="bebop_controller" type="bebop_controller" output="screen">
    <param name="frame" value="$(arg frame)" />
    <param name="worldFrame" value="$(arg worldFrame)" />
    <rosparam command="load" file="$(find bebop_controller)/config/bebop.yaml" />
  </node>
</launch>
